src/pages/permissions/PermissionIdentities.tsx

```diff
@@ -0,0 +1,293 @@
+    { content: "Auth method", sortKey: "authmethod" },
+    { content: "Type", sortKey: "type" },
+    {
+      content: "Groups",
+      sortKey: "groups",
+      className: "u-align--right",
+    },
+    { "aria-label": "Actions", className: "u-align--right actions" },
+  ];
+
+  const filters: PermissionIdentitiesFilterType = {
+    queries: searchParams.getAll(QUERY),
+    authMethod: searchParams.getAll(AUTH_METHOD),
+  };
+
+  const filteredIdentities = identities.filter((identity) => {
+    if (
+      !filters.queries.every(
+        (q) =>
+          identity.name.toLowerCase().includes(q) ||
+          identity.id.toLowerCase().includes(q),
+      )
+    ) {
+      return false;
+    }
+
+    if (
+      filters.authMethod.length > 0 &&
+      !filters.authMethod.includes(identity.authentication_method)
+    ) {
+      return false;
+    }
+
+    return true;
+  });
+
+  const selectedIdentities = identities.filter((identity) => {
+    if (panelParams.identity) {
+      return identity.id === panelParams.identity;
+    }
+
+    return selectedIdentityIds.includes(identity.id);
+  });
+
+  const identitiesForRows =
+    panelParams.panel === panels.identityGroups
+      ? selectedIdentities
+      : filteredIdentities;
+
+  const rows = identitiesForRows.map((identity) => {
+    const isLoggedInIdentity = settings?.auth_user_name === identity.id;
+    const isTlsIdentity = identity.authentication_method === "tls";
+    return {
+      name: isTlsIdentity ? "" : identity.id,
+      key: identity.id,
+      className: "u-row",
+      columns: [
+        {
+          content: (
+            <>
+              {identity.name} <Tag show={isLoggedInIdentity}>You</Tag>
+            </>
+          ),
+          role: "cell",
+          "aria-label": "Name",
+          className: "u-truncate",
+          title: identity.name,
+        },
+        {
+          content: identity.id,
+          role: "cell",
+          "aria-label": "ID",
+          className: "u-truncate",
+          title: identity.id,
+        },
+        {
+          content: identity.authentication_method.toUpperCase(),
+          role: "cell",
+          "aria-label": "Auth method",
+        },
+        {
+          content: identity.type,
+          role: "cell",
+          "aria-label": "Type",
+          className: "u-truncate",
+          title: identity.type,
+        },
+        {
+          content: identity.groups?.length || 0,
+          role: "cell",
+          className: "u-align--right",
+          "aria-label": "Groups for this identity",
+        },
+        {
+          content: !isTlsIdentity && (
+            <Button
+              appearance="base"
+              hasIcon
+              dense
+              onClick={() => panelParams.openIdentityGroups(identity.id)}
+              type="button"
+              aria-label="Manage groups"
+              title="Manage groups"
+            >
+              <Icon name="share" />
```

Comment:
We should better use the `user-group` icon here. The share is not expected.

Reply:
This is one of the design questions I'd like to ask @piperdeck. The `user-group` icon makes the most sense but it's also used for managing identities for a group, not sure if we want to differentiate the two actions with different icons, ultimately it's all about group membership for identities.

Reply:
My gut feeling is it is fine to reuse the icon as the two actions are very similar, just from different angles.

Reply:
Cool I have changed it to `user-group` for now